stable/wordpress/templates/deployment.yaml

```diff
@@ -19,15 +19,35 @@ spec:
         image: "{{ .Values.image }}"
         imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
         env:
+        - name: ALLOW_EMPTY_PASSWORD
+          value: {{ default "no" .Values.allowEmptyPassword | quote }}
```

Comment:
if user specifies `allowEmptyPassword` value without qoutes, go will treat this as a boolean value. User will HAVE TO specify the `allowEmptyPassword` with double quotes in values.yaml, eg. `allowEmptyPassword: "yes"`. I would not regard this as a stable UX. Instead i suggest using the following snippet. 

```go
        {{- if .Values.allowEmptyPassword }}
        - name: ALLOW_EMPTY_PASSWORD
          value: "yes"
        {{- end }}  
```

or 

```go
        - name: ALLOW_EMPTY_PASSWORD
        {{- if .Values.allowEmptyPassword }}
          value: "yes"
        {{- else }}
          value: "no"
        {{- end }} 
```